scrapyd/webservice.py

```diff
@@ -140,4 +143,5 @@ def render_POST(self, txrequest):
         project = txrequest.args['project'][0]
         version = txrequest.args['version'][0]
         self._delete_version(project, version)
+        UtilsCache.invalid_cache(project)
```

Comment:
This is a bit too broad. It could potentially invalidate only 1 or 2 cache entries but I don't think that performance justifies the extra code complexity.